code/modules/mob/living/carbon/human/species_types/psyphoza.dm

```diff
@@ -55,6 +55,12 @@
 		if(findname(.))
 			. = .(gender, TRUE, null, ++attempts)
 
+/datum/species/apid/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H)
```

Comment:
wrong species

Reply:
You can tell where I yoinked code from, lol. Thanks mate.